### PR TITLE
Added new style to make slider responsive

### PR DESCRIPTION
**Pull request by liudmylailchenko**

### CODE REVIEW

File: src/components/Card.js [hunk 1]
```diff
@@ -3,9 +3,9 @@ import styled from 'styled-components/macro';
 import { H3, TextSm } from './Typography';
 
 const Wrapper = styled.div`
-  flex: 0 0 33.333%;
+  flex: 0 0 100%;
   padding: 0 16px;
-
+  text-align: center;
   h3 {
     :hover {
```
Comment (zhukmj) [inline src/components/Card.js]: Please use ampersand

File: src/components/Card.js [hunk 2]
```diff
@@ -15,6 +15,18 @@ const Wrapper = styled.div`
...
+    max-width: 100%;
+  }
+
+  @media screen and (min-width: 500px) {
+    flex: 0 0 50%;
+  }
+
+  @media screen and (min-width: 770px) {
+    flex: 0 0 33.333%;
+  }
```
Comment (zhukmj) [inline src/components/Card.js]: These styles relate to Wrapper so it's better to move them up to Wrapper styles (above other selectors)

**Review by zhukmj — CHANGES_REQUESTED** [review]

File: src/components/Card.js (hunk 1 above)
Comment (liudmylailchenko) [reply src/components/Card.js]: Done

File: src/components/Card.js (hunk 2 above)
Comment (liudmylailchenko) [reply src/components/Card.js]: Done

**Review by zhukmj — APPROVED** [review]

**Pull request merged**